lib/gui/app/styled-components.tsx

```diff
@@ -134,24 +124,23 @@ const modalFooterShadowCss = css`
 	background-attachment: local, local, scroll, scroll;
 `;
 
-export const Modal = styled(({ style, ...props }) => {
+export const Modal = styled(({ style, children, ...props }) => {
 	return (
 		<Provider
-			theme={{
-				...defaultTheme,
+			theme={_.merge({}, theme, {
 				header: {
 					height: '50px',
 				},
 				layer: {
 					extend: () => `
-					${defaultTheme.layer.extend()}
+						${theme.layer.extend()}
 
-					> div:last-child {
-						top: 0;
-					}
-				`,
+						> div:last-child {
+							top: 0;
+						}
+					`,
 				},
-			}}
+			})}
```

Comment:
Please remove this.
We want one theme, we don't want to extend it in some places.
Remove the `<Provider>` tag and set
```
	header: {
		height: '40px',
	},
```
in `lib/gui/app/theme.ts`'s `theme`, you'll get the same result.